generators/api/index.js

```diff
@@ -1,168 +1,187 @@
+            return Object.assign({}, choice, {disabled: 'Accessible only by admin users'});
+          }
+
           return choice;
+
         });
       },
-      when: function () {
+      when () {
         return authMethods.length;
       }
-    }, {
+    },
+    {
       type: 'confirm',
       name: 'generateModel',
       message: 'Do you want to generate a model?',
       default: true
-    }, {
+    },
+    {
       type: 'input',
       name: 'modelFields',
       message: 'Which fields the model will have? (comma separated, do not include id)',
-      when: function (props) {
+      when (props) {
         return props.generateModel;
       }
-    }, {
+    },
+    {
       type: 'confirm',
       name: 'storeUser',
       default: true,
-      message: function (props) {
+      message (props) {
         return 'Do you want to store in a field the user who created the ' + props.kebab + '?';
       },
-      when: function (props) {
+      when (props) {
         return props.generateModel && props.userMethods && props.userMethods.indexOf('POST') !== -1;
       }
-    }, {
+    },
+    {
       type: 'input',
       name: 'userField',
       message: 'What\'s the name of the field which will store the user?',
       default: 'user',
-      when: function (props) {
+      when (props) {
         return props.storeUser;
       }
-    }, {
+    },
+    {
       type: 'confirm',
       name: 'getList',
       message: 'Do you want the retrieve methods (GET) to have the form { rows, count } ?',
       default: false,
-      when: function (props) {
-        var methods = getSelectedMethods(props);
-        return methods.find(function (method) {
+      when (props) {
+        const methods = getSelectedMethods(props);
+        
+        return methods.find((method) => {
           return method.value === 'GET LIST';
         }) && props.generateModel;
       }
     }];
 
-    return this.prompt(prompts).then(function (props) {
+    return this.prompt(prompts).then((props) => {
       this.props = props;
-      this.props.camel = _.camelCase(this.props.kebab);
-      this.props.camels = pluralize(this.props.camel);
-      this.props.pascal = _.upperFirst(this.props.camel);
-      this.props.pascals = _.upperFirst(this.props.camels);
-      this.props.lower = _.lowerCase(this.props.camel);
-      this.props.lowers = _.lowerCase(this.props.camels);
-      this.props.start = _.upperFirst(this.props.lower);
-      this.props.starts = _.upperFirst(this.props.lowers);
+      this.props.camel   = camelCase(this.props.kebab);
+      this.props.camels  = pluralize(this.props.camel);
+      this.props.pascal  = upperFirst(this.props.camel);
+      this.props.pascals = upperFirst(this.props.camels);
+      this.props.lower   = lowerCase(this.props.camel);
+      this.props.lowers  = lowerCase(this.props.camels);
+      this.props.start   = upperFirst(this.props.lower);
+      this.props.starts  = upperFirst(this.props.lowers);
```

Comment:
Does that pass eslint? I don't think we need those alignments.

Reply:
we can create our custom rules... 
Currently i am using eslint recommend, but we could use another rules about indentation, spaces, ect..